### PR TITLE
Update + add more jmx receiver integration tests.

### PR DESCRIPTION
**Pull request by jsuereth**
- Add JMX test for latest JMX jar release
- Update integration tests to check multiple versions of JMX jar

**Description:** 
Expand JMX receiver integration tests to included latest release.

**Testing:** 
Ran locally (only adds tests).

**Documentation:** 
N/A.  (Note: there will be additions on java-contrib repository denoting this code and making sure it gets updates on release).

### CODE REVIEW

File: receiver/jmxreceiver/integration_test.go [hunk 1]
```diff
@@ -126,117 +134,123 @@ func getLogsOnFailure(t *testing.T, logObserver *observer.ObservedLogs) {
...
+				require.True(t, ok)
+				require.Equal(t, "myvalue", customAttr.StringVal())
+
+				anotherCustomAttr, ok := attributes.Get("myotherattr")
+				require.True(t, ok)
+				require.Equal(t, "myothervalue", anotherCustomAttr.StringVal())
+
+				ilm := rm.InstrumentationLibraryMetrics().At(0)
+				require.Equal(t, "io.opentelemetry.contrib.jmxmetrics", ilm.InstrumentationLibrary().Name())
+				require.Equal(t, "1.0.0-alpha", ilm.InstrumentationLibrary().Version())
```
Comment (rmfitzpatrick) [inline receiver/jmxreceiver/integration_test.go:227]: This needs to change in the gatherer so will eventually be an issue.

**Review by rmfitzpatrick — APPROVED** [review]

**Review by djaglowski — APPROVED** [review]
LGTM - Test failure appears to be unrelated.

File: receiver/jmxreceiver/integration_test.go (hunk 1 above)
Comment (jsuereth) [reply receiver/jmxreceiver/integration_test.go:227]: Oh my, I didn't even notice.  That'll be a fun one.

Comment (rmfitzpatrick) [reply receiver/jmxreceiver/integration_test.go:227]: https://github.com/open-telemetry/opentelemetry-java-contrib/pull/77

File: receiver/jmxreceiver/integration_test.go [hunk 2]
```diff
@@ -43,25 +43,33 @@ import (
...
 
 func TestJMXIntegration(t *testing.T) {
 	suite.Run(t, new(JMXIntegrationSuite))
 }
 
 func (suite *JMXIntegrationSuite) SetupSuite() {
-	jarPath, err := downloadJMXMetricGathererJAR()
-	require.NoError(suite.T(), err)
-	suite.JARPath = jarPath
+	jmxJarReleases := map[string]string{
```
Comment (anuraaga) [inline receiver/jmxreceiver/integration_test.go:54]: Can you move these up to a top level constant so they're most obvious for someone updating after a java-contrib release?

Comment (jsuereth) [reply receiver/jmxreceiver/integration_test.go]: Fixed

**Review by anuraaga — APPROVED** [review]

**Pull request merged**